pinot-broker/src/main/java/com/linkedin/pinot/broker/routing/builder/BaseRoutingTableBuilder.java

```diff
@@ -86,10 +148,52 @@ protected void setRoutingTables(List<Map<String, List<String>>> routingTables) {
     return _routingTables;
   }
 
-  protected void handleNoServingHost(String segmentName) {
-    LOGGER.error("Found no server hosting segment {} for table {}", segmentName, _tableName);
```

Comment:
We should keep this error log (this means 0 replica up), but do not call this method in computeDynamicRoutingTable()

Reply:
fixed